Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -2502,6 +2502,8 @@ class RenderStyle final : public CanMakeCheckedPtr<RenderStyle, WTF::DefaultedOp
     inline bool insideSubmitButton() const;
     inline void setInsideSubmitButton(bool);
 
+    bool hasDisplayTransition() const;
```

Comment:
It would be generally better to not pile up logic into `RenderStyle`. Perhaps this could be a standalone function somewhere in the animation code?